metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -190,3 +190,16 @@ spec:
       for: 15s
       labels:
         severity: critical
+  - name: ocs-encryption-alert.rules
+    rules:
+    - alert: KMSServerConnection
+      annotations:
+        description: Storage Cluster KMS Server is in un-connected state for more than 5s. Please check KMS config.
+        message: Storage Cluster KMS Server is in un-connected state. Please check KMS conifg.
+        severity_level: error
+        storage_type: ceph
+      expr: |
+        ocs_storagecluster_kms_connection_status{job="ocs-metrics-exporter"} == 1
+      for: 5s
```

Comment:
I gave 5 seconds because that's the timeout we have in the code to check if KMS is reachable or not, [here](https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/cephcluster.go#L177)

Reply:
But again, that's for the initial connection so for generating alerts maybe we can keep it as 10s.